### PR TITLE
NOISSUE - Remove authz from docker comp

### PR DESCRIPTION
**Pull request by mteodor**
### What does this do?
Remove `authz` service from docker composition as `authz` functionality will be merged with `authn` service

### CODE REVIEW

File: users/postgres/init.go [hunk 1]
```diff
@@ -96,6 +96,13 @@ func migrateDB(db *sqlx.DB) error {
 				)`,
 				},
 			},
+			{
+				Id: "users_5",
+				Up: []string{
+					`ALTER TABLE IF EXISTS users DROP CONSTRAINT IF EXISTS users_owner_id_fkey`,
+					`ALTER TABLE IF EXISTS users DROP COLUMN IF EXISTS owner_id`,
+				},
+			},
```
Comment (manuio) [inline users/postgres/init.go]: Dead migration?

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by nmarcetic — CHANGES_REQUESTED** [review]
@mteodor  You still have 3 commits. Is there is a special reason why you are removing docker-compose stuff before `authz` is merged in `authn`? Does this will break functionality?

**Comment by mteodor** [comment]
> @mteodor You still have 3 commits. Is there is a special reason why you are removing docker-compose stuff before `authz` is merged in `authn`? Does this will break functionality?

authz is not used, and there is a problem with starting the service in docker composition so that is why I removed it
authz was not complete since it was the idea to break it in few PR's
in a meanwhile we decided to have authn and authz in one service

**Comment by nmarcetic** [comment]
@mteodor I see. Thanks for the clarification. 
Maybe is a good idea to remove it from all builds like Makefile (build, push etc...) and leave just source code for now.

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**